tests/components/vodafone_station/test_config_flow.py

```diff
@@ -216,3 +209,31 @@ async def test_reauth_not_successful(hass: HomeAssistant, side_effect, error) ->
 
         assert result2["type"] == FlowResultType.ABORT
         assert result2["reason"] == "reauth_successful"
+
+
+async def test_options_flow(hass: HomeAssistant) -> None:
+    """Test options flow."""
+
+    mock_config = MockConfigEntry(domain=DOMAIN, data=MOCK_USER_DATA)
+    mock_config.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.vodafone_station.config_flow.VodafoneStationSercommApi.login",
+    ), patch(
+        "homeassistant.components.vodafone_station.config_flow.VodafoneStationSercommApi.logout",
+    ), patch(
+        "homeassistant.components.vodafone_station.async_setup_entry",
+    ):
+        result = await hass.config_entries.options.async_init(mock_config.entry_id)
+        assert result["type"] == FlowResultType.FORM
+        assert result["step_id"] == "init"
+
+        result = await hass.config_entries.options.async_init(mock_config.entry_id)
```

Comment:
Why are we starting the flow twice?

Reply:
Barely copy&paste from `fritz` ;-)
Fixed.